src/components/TopBar/TopBarMenu.vue

```diff
@@ -84,62 +136,37 @@
 					{{ t('spreed', 'Stop recording') }}
 				</NcActionButton>
 			</template>
-			<!-- Device settings -->
+
+			<!-- Breakout rooms -->
 			<NcActionButton :close-after-click="true"
-				@click="showSettings">
+				@click="openBreakoutRoomsEditor">
 				<template #icon>
-					<VideoIcon :size="20" />
+					<DotsCircle :size="20" />
 				</template>
-				{{ t('spreed', 'Devices settings') }}
+				{{ t('spreed', 'Setup breakout rooms') }}
```

Comment:
Let's only have the "start/stop" here in the menu. The configuration is in the conversation settings or later on in the right sidebar tab